packages/shared/src/contexts/BootProvider.tsx

```diff
@@ -85,6 +84,14 @@ const updateLocalBootData = (
   return result;
 };
 
+const getCachedOrNull = () => {
+  try {
+    return JSON.parse(storage.getItem(BOOT_LOCAL_KEY));
```

Comment:
@capJavert parse would fail if the cache is empty, wrapped it in a try-catch to handle the case and return as intended.